lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.h

```diff
@@ -964,6 +964,7 @@ class SwiftASTContext : public TypeSystemSwift {
   bool m_initialized_language_options = false;
   bool m_initialized_search_path_options = false;
   bool m_initialized_clang_importer_options = false;
+  bool m_explicit_modules = false;
```

Comment:
Nit: from the variable's name I'd expect this to be a vector of explicit modules. Could this be `m_contains_explicit_modules` or something like that?